ubyssey/static/src/styles/main.scss

```diff
@@ -88,6 +88,14 @@ body, html, #page-wrapper {
   -webkit-overflow-scrolling: touch;
 }
 
+span.upper-right-decorator {
+  color: #006DCC;
```

Comment:
This can be replaced with `$color-ub-blue-light`

Reply:
yea